com.github.sdbg.debug.ui/src/com/github/sdbg/ui/ISDBGLauncher.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui;
+
+import org.eclipse.core.resources.IResource;
+
+public interface ISDBGLauncher {
+
+  String EXTENSION_ID = "com.github.sdbg.debug.ui.sdbgLauncher";
+
+  public void launchChrome(IResource resource, String mode, String url);
```

Comment:
Ok, well, it's up to us to pick the most maintainable API possible.  If we can add more options to the launcher to do more integration at other levels, then we should definitely keep them separate.  If this is a one-time thing, I would personally prefer to adapt existing code than to add a new shell class.

At the end of the day, so long as it works, I'm happy :-)